lib/embeds/instagram.js

```diff
@@ -18,7 +18,16 @@ export default {
       : '';
 
     const script = `<script src="${protocol}//platform.instagram.com/en_US/embeds.js"></script>
-      <script>resize()</script>`;
+      <script>
+        const interval = setInterval(function () {
+          const iframe = document.querySelector('iframe.instagram-media-rendered');
+          if (iframe && iframe.height) {
+            resize();
+            clearInterval(interval);
+          }
+        }, 100);
```

Comment:
@ryanscottaudio It could be some other interval, but 100 seemed like a reasonable number here. We can't use DOMContentLoaded because we're waiting for the embed to be completely rendered, and there's no callback for that in the sdk api.